src/Solver/Interoperability.cpp

```diff
@@ -1184,16 +1216,16 @@ void seissol::Interoperability::getDynRupTP(real TP_grid[seissol::dr::TP_grid_nz
   f_interoperability_getDynRupTP(m_domain,  &TP_grid[0], &TP_DFinv[0]);
 }
 
-void seissol::Interoperability::copyFrictionOutputToFortranGeneral(unsigned ltsFace, unsigned meshFace,
-                                                                   real  (*slip)[init::QInterpolated::Stop[0]],
-                                                                   real  (*slipStrike)[init::QInterpolated::Stop[0]],
-                                                                   real  (*slipDip)[init::QInterpolated::Stop[0]],
-                                                                   real  (*ruptureTime)[init::QInterpolated::Stop[0]],
-                                                                   real  (*dynStressTime)[init::QInterpolated::Stop[0]],
-                                                                   real  (*peakSlipRate)[init::QInterpolated::Stop[0]],
-                                                                   real  (*tractionXY)[init::QInterpolated::Stop[0]],
-                                                                   real  (*tractionXZ)[init::QInterpolated::Stop[0]]
-        ){
+void seissol::Interoperability::copyFrictionOutputToFortranGeneral(
```

Comment:
seems you used a const expr for init::QInterpolated::Stop[0] somewhere else in the code.

Reply:
Added 
```
template <typename Tensor>
constexpr size_t leadDim() noexcept {
  return Tensor::Stop[0] - Tensor::Start[0];
}
static constexpr inline size_t AlignedNumGaussianPoints = leadDim<init::QInterpolated>();
```
to `Misc.h`

I replaced all occurrences of `init::QInterpolated::Stop[0]` with `AlignedNumGaussianPoints`

Reply:
good!